### PR TITLE
Changelog and readme updates FormRecognizer

### PR DESCRIPTION
**Pull request by samvaity**
Changelog updates following the guidance here - https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change

Readme updates - using absolute links over relative links and metadat update

Closes : #11036

### CODE REVIEW

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,23 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-06-10)
+### New Features
+- Support to copy a custom model from one Form Recognizer resource to another
+- Added support for AAD Authentication.
+- Raise `FormRecognizerException` when a model with `ModelStatus.Invalid` is returned from the `beginTraining()` API's
+- Raise `FormRecognizerException` when invalid analyze status returned from service for recognize API's
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:8]: incomplete sentense?

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 2]
```diff
@@ -16,25 +26,24 @@ must now be extracted from its `RecognizedForm`.
...
-### New Features
-- Support to copy a custom model from one Form Recognizer resource to another
+### Key Bug Fixes
+- Fix bug in FormRecognizer API's to support multipage document recognition.
+- Fix Receipt type to select the valueString field via fieldValue.
+
+This package's
+[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-ai-formrecognizer_1.0.0-beta.3/sdk/formrecognizer/azure-ai-formrecognizer/README.md)
+and
+[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-ai-formrecognizer_1.0.0-beta.3/sdk/formrecognizer/azure-ai-formrecognizer/src/samples)
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:43]: is `azure-ai-formrecognizer_1.0.0-beta.3` goingt to be created after release?

File: sdk/formrecognizer/azure-ai-formrecognizer/README.md [hunk 3]
```diff
@@ -430,40 +430,40 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 [coc_faq]: https://opensource.microsoft.com/codeofconduct/faq/
 [coc_contact]: mailto:opencode@microsoft.com
 [create_new_resource]: https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account?tabs=multiservice%2Cwindows#create-a-new-azure-cognitive-services-resource
-[differentiate_custom_forms_with_labeled_and_unlabeled_models]: src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledData.java
+[differentiate_custom_forms_with_labeled_and_unlabeled_models]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer//AdvancedDiffLabeledUnlabeledData.java
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/README.md:433]:
```suggestion
[differentiate_custom_forms_with_labeled_and_unlabeled_models]: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledData.java
```

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 4]
```diff
@@ -1,13 +1,23 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
+## 1.0.0-beta.3 (2020-06-10)
```
Comment (maririos) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:3]: my understanding is that we are releasing on 06/09, no?

File: sdk/formrecognizer/azure-ai-formrecognizer/src/samples/README.md [hunk 5]
```diff
@@ -5,6 +5,7 @@ languages:
 products:
   - azure
   - azure-cognitive-service
```
Comment (mssfang) [inline sdk/formrecognizer/azure-ai-formrecognizer/src/samples/README.md:7]:
```suggestion
  - azure-cognitive-services
```

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 1 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md]: updated it a little bit

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 6]
```diff
@@ -1,13 +1,23 @@
...
+### New Features
+- Support to copy a custom model from one Form Recognizer resource to another
+- Added support for AAD Authentication.
+- Raise `FormRecognizerException` when a model with `ModelStatus.Invalid` is returned from the `beginTraining()` API's
+- Raise `FormRecognizerException` when an invalid analyze status is returned from the service for recognize API's
+- Add `pageNumber` property to `FormPage` and `FormTable` model
+- Update FormRecognizer API calls to return a `List` instead of `IterableStream`.
```
Comment (maririos) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:10]: will this be a breaking change? (not sure Java...)

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 4 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:3]: I wasn't sure of the date so much. 
I am happy to change. Was it TA on 06/09?

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md [hunk 7]
```diff
@@ -1,13 +1,23 @@
...
+- Add `getFormRecognizerClient()` and `getFormRecognizerAsyncClient()` in FormTrainingClient and FormTrainingAsyncClient
+- Add `FormTrainingClientBuilder` to build `FormTrainingAsyncClient` and `FormTrainingClient`
+- Adopt the `training` namespace for Form Recognizer Training Clients
```
Comment (maririos) [inline sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:13]: will this be a breaking change compared to previous version?

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 4 above)
Comment (mssfang) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:3]: yes. TA is 06/09

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 6 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:10]: yes!

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 7 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:13]: and yes!

File: sdk/formrecognizer/azure-ai-formrecognizer/README.md (hunk 3 above)
Comment (maririos) [inline sdk/formrecognizer/azure-ai-formrecognizer/README.md:433]: consider removing extra `/` in `formrecognizer//AdvancedDiffLabeledUnlabeledData.java`

File: sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md (hunk 4 above)
Comment (samvaity) [reply sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md:3]: I will keep this as 06/10 for now. I will update if we decide otherwise!

**Pull request merged**